docs/core/additional-tools/uninstall-tool.md

```diff
@@ -464,7 +464,7 @@ Notes:
 > [!NOTE]
 > By default, .NET Core SDKs and runtimes that may be required by Visual Studio or other SDKs are kept. In the following examples, some of the specified SDKs and runtimes may remain, depending on the state of the machine. To remove all SDKs and runtimes, list them explicitly as arguments or use the `--force` option.
 
-* Remove all x86 .NET Core runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/n confirmation:
+* Remove all .NET Core runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/n confirmation:
```

Comment:
How does the confirmation works? Wondering if we could say here yes/no confirmation.

Reply:
The user is asked to enter `Y` for confirmation, and right now anything besides that (including `yes`) is taken as no. I plan on changing this but for right now that's the situation.

Reply:
Ok. Maybe we should just capitalize like this then?

```suggestion
* Remove all .NET Core runtimes except the version `3.0.0-preview6-27804-01` without requiring Y/N confirmation:
```